docs/snapshots.md

```
@@ -63,20 +63,20 @@ The `<id>` in `settings.xml` must match the `<id>` of the `<repository>` block i
 </dependency>
 ```
 
 Replace the version with whichever in-progress snapshot you want. Browse available versions: https://github.com/orgs/kzmlabs/packages?repo_name=flink-statefun
 
 ## Maintainer flow — publishing a new snapshot
 
-1. Bump the `<revision>` in root `pom.xml` to a `-SNAPSHOT` version (e.g. `3.4.0-KZM-3.4-SNAPSHOT`)
+1. Bump the `<revision>` in root `pom.xml` to a `-SNAPSHOT` version (e.g. `3.4.0-KZM-3.5-SNAPSHOT`)
 2. Merge that bump to `release` branch
 3. Tag and push:
    ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)